### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by harshkotkar5557**
Preview Image
user-profile

### CODE REVIEW

File: user-profile/user-profile.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    max-height: 65vh;
+}
+.order-summary-container{
+    overflow: auto;
+    height: 40vh;
+ 
+}
+/* .order-summary-container::-webkit-scrollbar{
+    display: none;
+} */
```
Comment (aamrapaliwandhre) [inline user-profile/user-profile.css:42]: remove this comment..

**Review by aamrapaliwandhre — APPROVED** [review]
looks good to me...

File: user-profile/profile.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+            </ul>
+        </header>
+    </nav>
+    
+    <section class="middleSection login-middle d-flex align-center justify-center">
+            <div class="card user-info-profile-container p-1 w-half">
+                    <input type="radio" name="sub" id="profile" checked>
+                    <input type="radio" name="sub" id="address">
+                    <input type="radio" name="sub" id="orders">
+                    <input type="radio" name="sub" id="setting">
```
Comment (vaibhav18matere) [inline user-profile/profile.html:57]: try to use descriptive meanings to `names`

**Review by vaibhav18matere — APPROVED** [review]

**Review by Mahi24-prog — APPROVED** [review]
Looks good go ahead

**Pull request merged**